CONTRIBUTION.md

```diff
@@ -0,0 +1,78 @@
+# How to contribute
+
+Windows Machine Config Operator is Apache 2.0 licensed and accepts contributions via GitHub pull requests. This
+document outlines some of the conventions on commit message formatting and other resources to help get contributions into the project.  
+
+
+## Reporting bugs and creating issues
+
+If any part of the project has bugs or documentation mistakes, please let us know by opening a
+[Jira issue](https://jira.coreos.com/projects/WINC/summary)
+
+## Contribution flow
+
+This is an outline of what a contributor's workflow looks like:
+
+- Fork the repository on GitHub.
+- Clone the forked repository outside your go path, or refer to the [Kubernetes repo's example](https://github.com/kubernetes/community/blob/master/contributors/guide/github-workflow.md#2-clone-fork-to-local-storage).
+- Create a topic branch, branched off of master.
+- Make commits of logical units. A commit should typically add a feature or fix a bug, but never both at the same
+time. Vendor commits should always be separate.
+- A PR should consist of a set of logically connected commits to the issue that makes it easy to review and should not 
+follow your personal development flow.
+- Make sure commit messages are in the proper format (see below).
+- Push changes in a topic branch to a personal fork of the repository.
+- To make sure that your topic branch is in sync with the remote master branch,
+follow a [rebase workflow](https://www.atlassian.com/git/tutorials/merging-vs-rebasing).
+- Submit a pull request to openshift/windows-machine-config-operator (see PR workflow below).
+- The PR must receive one `/lgtm` and one `/approve` comments from the maintainers of the project.
+
+Thanks for contributing!
+
+### Format of the commit message
+
+We follow a convention for commit messages that is designed to answer two questions: what changed and why. The
+subject line should feature the what and the body of the commit should describe the why.
+
+The format can be described more formally as follows:
+
+```
+[subsystem] <what changed>
+<BLANK LINE>
+<why this change was made>
+<BLANK LINE>
+<footer>
+```
+Example for a sample feature:
+```
+[docs] Add the Guidlines
+
+Cupcake ipsum dolor sit. Amet tart cheesecake tiramisu chocolate cake topping.
+Icing ice cream sweet roll. Biscuit dragée toffee wypas.
+Follow-up to Id5e7cbb1.
+```
+
+The first line is the subject and should be no longer than 50 characters, the second line is always blank, and other
+lines should be wrapped at 80 characters. This allows the message to be easier to read on GitHub as well as in various
+git tools. If it is a bug ticket, the bug number should be mentioned at the start of the subject.
```

Comment:
Please give an example. No need for a full example but something like `Example - Bug 123: Bug fix description`